src/components/product.js

```diff
@@ -396,7 +396,7 @@ export default class Product extends Component {
         values: option.values.map((value) => {
           return {
             name: value.value,
-            selected: Object.values(this.selectedOptions).some((selectedOption) => {
+            selected: Object.values([this.selectedOptions[option.name]]).some((selectedOption) => {
```

Comment:
Object.values on an array returns that array. I think this is calling `some` on an array of one item. I think n this call is equivalent to `this.selectedOptions[option.name] === value.value`